Src/Base/AMReX_GpuLaunchFunctsC.H

```diff
@@ -24,6 +24,16 @@ namespace detail {
         f(i, Gpu::Handler{});
     }
 
+    // call_f_vector_handler
+
+    template <int SIMD_WIDTH, typename F, typename N>
+    AMREX_FORCE_INLINE
+    auto call_f_vector_handler (F const& f, N i)
+        noexcept -> decltype(f(0))
```

Comment:
```suggestion
        noexcept -> decltype(f.template operator()<SIMD_WIDTH>(0))
```
Also there should be a second version of this with Gpu::Handler{} to match the scalar version.

Reply:
Thx for the fix!

> Also there should be a second version of this with Gpu::Handler{} to match the scalar version.

Yes, def worth for some GPU kernels, but needs extra work. So far I only use this in CPU code.

Reply:
I that case, maybe rename it to `call_f_vector` or `call_f_simd`